src/Octoshift/Services/GithubApi.cs

```diff
@@ -803,7 +803,10 @@ public virtual async Task<ReattributeMannequinToUserResult> ReclaimMannequinsSki
         var response = await _client.PostAsync(url, payload);
         var data = JObject.Parse(response);
 
-        return data.ToObject<ReattributeMannequinToUserResult>();
+        return data["errors"].Any(x => ((string)x["message"]).Contains("Field 'reattributeMannequinToUser' doesn't exist on type 'Mutation'"))
```

Comment:
I would call `_client.PostGraphQLAsync()` instead of `PostAsync()`, it will handle parsing the GQL response body and checking if there are any errors. Then you'd just need to do a try/catch and check the error message and change it and rethrow.

Reply:
Good point: updated[ here](9dfd63e535ffb35b4434178b2ab6c7a7a89e1a94)